src/test/java/uk/gov/pay/adminusers/resources/InviteResourceServiceCompleteIT.java

```diff
@@ -57,7 +62,30 @@ void shouldReturn200withDisabledInviteLinkingToCreatedUser_WhenPassedAValidInvit
     }
 
     @Test
-    void shouldReturn410_WhenSameInviteCodeCompletedTwice() {
+    void shouldReturn400_whenUnsupportedSecondFactorMethodSupplied() throws Exception {
+        String email = format("%s@example.gov.uk", randomUuid());
+        String username = email;
+        String telephoneNumber = "+447700900000";
+        String password = "valid_password";
+
+        Map<String, String> payload = Map.of("second_factor", "BAD_VALUE");
+
+        String inviteCode = inviteDbFixture(databaseHelper)
+                .withTelephoneNumber(telephoneNumber)
+                .withEmail(email)
+                .withPassword(password)
+                .insertServiceInvite();
```

Comment:
This test shouldn't need to insert an invite into the database as it shouldn't get that far. Maybe worth removing this to avoid any confusion.